src/openfermion/utils/_remove_symmetry_qubits_test.py

```diff
@@ -0,0 +1,171 @@
+        h2_sto_hamil, h2_sto_numorb, h2_sto_numel = H2_sto3g()
+        h2_631_hamil, h2_631_numorb, h2_631_numel = H2_631g()
+        lih_sto_hamil, lih_sto_numorb, lih_sto_numel = LiH_sto3g()
+
+        # Use test function to reduce the qubits.
+        h2_sto_qbt = _remove_symmetry_qubits.remove_symmetry_qubits(h2_sto_hamil,
+                                                                 h2_sto_numorb,
+                                                                 h2_sto_numel)
+        h2_631_qbt = _remove_symmetry_qubits.remove_symmetry_qubits(h2_631_hamil,
+                                                                 h2_631_numorb,
+                                                                 h2_631_numel)
+        lih_sto_qbt = _remove_symmetry_qubits.remove_symmetry_qubits(lih_sto_hamil,
+                                                                  lih_sto_numorb,
+                                                                  lih_sto_numel)
+        
+        self.assertAlmostEqual(eigenspectrum(h2_sto_qbt)[0],
+                         eigenspectrum(h2_sto_hamil)[0])
+
+        self.assertAlmostEqual(eigenspectrum(h2_631_qbt)[0],
+                         eigenspectrum(h2_631_hamil)[0])
+
+        self.assertAlmostEqual(eigenspectrum(lih_sto_qbt)[0],
+                         eigenspectrum(lih_sto_hamil)[0])
+
+
+
+
+    # Check that the qubit Hamiltonian acts on two fewer qubits.
+    def test_orbnum_reduce_symmetry_qubits(self):
+        # Generate the fermionic Hamiltonians,
+        # number of orbitals and number of electrons.
+        h2_sto_hamil, h2_sto_numorb, h2_sto_numel = H2_sto3g()
+        h2_631_hamil, h2_631_numorb, h2_631_numel = H2_631g()
+        lih_sto_hamil, lih_sto_numorb, lih_sto_numel = LiH_sto3g()
+
+        # Use test function to reduce the qubits.
+        h2_sto_qbt = _remove_symmetry_qubits.remove_symmetry_qubits(h2_sto_hamil,
+                                                                 h2_sto_numorb,
+                                                                 h2_sto_numel)
+        h2_631_qbt = _remove_symmetry_qubits.remove_symmetry_qubits(h2_631_hamil,
+                                                                 h2_631_numorb,
+                                                                 h2_631_numel)
+        lih_sto_qbt = _remove_symmetry_qubits.remove_symmetry_qubits(lih_sto_hamil,
+                                                                  lih_sto_numorb,
+                                                                  lih_sto_numel)
+        
+        self.assertEqual(number_of_qubits(h2_sto_qbt, h2_sto_numorb),
+                         h2_sto_numorb-2)
+
+        self.assertEqual(number_of_qubits(h2_631_qbt, h2_631_numorb),
+                         h2_631_numorb-2)
+
+        self.assertEqual(number_of_qubits(lih_sto_qbt, lih_sto_numorb),
+                         lih_sto_numorb-2)
+
+
+if __name__ == '__main__':
```

Comment:
we do not have __main__ in any of our files. The tests are run through the pytest module.